frontend/model/chatroom/vuexModule.js

```diff
@@ -107,10 +107,14 @@ const getters = {
       })
     }
   },
-  isDirectMessage (state, getters) {
+  isGroupDirectMessage (state, getters) {
     // NOTE: identity contract could not be synced at the time of calling this getter
     return chatRoomID => !!getters.ourGroupDirectMessages[chatRoomID || getters.currentChatRoomId]
   },
+  isDirectMessage (state, getters) {
+    // NOTE: identity contract could not be synced at the time of calling this getter
+    return chatRoomID => !!getters.ourDirectMessages[chatRoomID || getters.currentChatRoomId]
+  },
```

Comment:
This is based on getter `ourDirectMessages` which I'm noticing is defined twice (DRY violation) in both `state.js` and `shared/getters/identity.js`.

Could you please delete the copy of it in `state.js`?

Reply:
Nice catch! I will remove it